jsk_rosbag_tools/requirements.in

```diff
@@ -1 +1,3 @@
 moviepy==1.0.3
+scipy<1.9.0
+numpy
```

Comment:
please also specity numpy version

Reply:
imo,
we dont need to specify numpy version, but yes.

Reply:
I change my mind and specify `numpy>=1.21.0` for `python3`.